keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -282,102 +319,145 @@ not always required) or even code snippets. If there's any ambiguity
 about HOW your proposal will be implemented, this is the place to
 discuss them. -->
 
-We will expose a endpoint `/discovery` that will support JSON, and
-protobuf and gzip compression. The endpoint will serve the aggregated
-discovery document for all types that a Kubernetes cluster supports.
+The current discovery endpoints `/api` and `/apis` will accept a new
+content negotiation type `Accept: as=APIGroupDiscoveryList;v=v1beta1;g=meta.k8s.io` that represents an
```

Comment:
How do you say that you accept the "former type", how will this look like when we have multiple versions?

Reply:
If there are multiple versions, the name and group should remain the same, the `v=<version>` field will change depending on what is accepted. I'm not sure what you mean by the other question.

Reply:
> How do you say that you accept the "former type"

Say your client doesn't support the un-aggregated form, what would the header look like? Or if it only supports un-aggregated, or both? What would each accept headers look like in these scenarios?

> how will this look like when we have multiple versions?

If you support both v1beta1 and v1, what will the header look like, can you give an example? 

How do I accept protobuf? json?

Reply:
Updated to include info around those